src/scripts/main.js

```diff
@@ -1,9 +1,27 @@
 'use strict';
 
+const htmlBody = document.body;
```

Comment:
you are using `htmlBody` only once, so just write `document.body` right away to not reproduce a lot of variables